_posts/2020-11-04-ggplot2.md

```diff
@@ -0,0 +1,92 @@
+
+# 2. Box and Whisker Plots
+
+Box and whisker plots are especially useful for scientific studies: it allows you not to just see data, but also the margin of error and how wide it is. A lot of parameters here
+are the same for *geom_col* and *geom_bar*, but *geom_boxplot* will need further elaboration about the lower and upper quartiles of data. Again, let's do this in a code
+chunk:
+
+```{r box-and-whisker-plot}
+#include default parameters here
+ggplot(nice_data, aes(x = "dependent value", y = "independent value")) +
+geom_boxplot(
+  mapping = NULL,
+  data = NULL,
+  stat = "boxplot",
+  position = "dodge2",
+  na.rm = FALSE,
+  orientation = NA,
+  show.legend = NA,
+  inherit.aes = TRUE,
+  outlier.color = NULL,
+  outlier.fill = NULL,
+  outlier.shape = 19,
+  outlier.size = 1.5,
+  outlier.stroke = 0.5,
+  outlier.alpha = NULL,
+  notch = FALSE,
+  notchwidth = 0.5,
+  varwidth = FALSE,
+  )
+```
+The outlier parameters refer to data that falls outside the box plot: you can customize the shape, size, and fill (if applicable) to your liking so they stand out from the rest of your dataset. If you do not want any outliers to show up, set **outlier.shape** to NA and they will disappear. Another useful aspect is the **notch** in the box plot that shows the median value of the data with a roughly 95% confidence interval. If **notch** is set to *TRUE*, it will show that notch and depending on **notchwidth**, you will see the width of the box getting smaller (think of an hourglass shape) and be smallest at where the median value is. **varwidth** refers to the width of the box: if set to *FALSE*, it will produce a standard box plot, and if set to *TRUE*, it will produce a box with the width proportional to the square root of number of observations (wider if there's more data).
+
```

Comment:
Maybe consider including a short conclusion / summary? The post ends rather abruptly.

Reply:
Hi @sarah-gao, I made the changes you requested, thanks for the feedback!